mms/tests/integration_tests/onnx_export_integ_test.py

```diff
@@ -15,6 +16,30 @@
 from mms import mxnet_model_server
 
 
+
+
+### models from onnx-mxnet model zoo
+onnx_mxnet_model_URLs = {
+    'onnx-alexnet'    : 'https://s3.amazonaws.com/model-server/models/onnx-alexnet/alexnet.onnx',
+    'onnx-squeezenet' : 'https://s3.amazonaws.com/model-server/models/onnx-squeezenet/squeezenet.onnx',
+    'onnx-inception_v1' : 'https://s3.amazonaws.com/model-server/models/onnx-inception_v1/inception_v1.onnx',
+    'onnx-vgg19' : 'https://s3.amazonaws.com/model-server/models/onnx-vgg19/vgg19.onnx'
+}
+
+
+### models from onnx model zoo
+onnx_model_URLs = {
```

Comment:
Sure I will create a separate issue to track this. 
One probable problem which I can foresee is onnx model zoo don't give signature files and we are using the one for alexnet for all image classification models as they work with same signature. This can become issue when the model uploaded in onnx comes from NLP or maybe require a different signature altogether.  Here you will need to manually create signature for that particular model and until its done, the tests and build will fail.

Reply:
Should we create dedicated model archive files for MMS integration tests?
This can help us decouple from ONNX and other repos that may update/remove their files without notice.

Reply:
@lupesko I have created a separate issue for the above issue which we can revisit.